### ISSUE TITLE
rename "food component" to "food component role"

### ISSUE THREAD

**Issue by ddooley** [ebi-chebi/ChEBI#3814]
At moment there are several roles in CHEBI that by their labels look like material entities.  Could they get "role" as a suffix?  This particular case collides with FoodOn's "food component" which is a material entity (which could have role 'food component role').

The CHEBI "food component" definition should be tweaked to indicate that it is a role, not a substance.  Alternately the term finds a home in the material entity branch?

d.

**Comment by amalik01** [comment]
The ontology clearly says that food component is a physiological role so what's the point in having role as a suffix?
To change 'food component' to 'food component role' will sound strange and the word 'role' would be duplicated. Lets take the child class piperine (CHEBI:28821) as an example. If you look at the ontology of this entry, its says 'has role food component' so the ontology clearly saying that its a role class rather than a material entity. If i add role as a suffix, it will then read 'has role food component role'.

**Comment by amalik01** [comment]
I agree with you with regards to the ChEBI definitions. I have therefore updated the ChEBI definitions for food (CHEBI:33290) and food component (CHEBI:78295) so that the user knows that these are role classes. If there are other entries in ChEBI where the definitions require updating, just send me the list and i will update them. Thanks

**Comment by ddooley** [comment]
I appreciate those changes to the definitions.  

I'd say the remaining problem from a text-mining (i.e. language comprehension) perspective is whether labels in the ChEBI "food component" class and subclasses (that appear to be defined with respect to parent term but don't mention "role" again) yield the most common semantic sense of the label?  If "vitamin" appears in a research paper is it a reference to a group of chemical having the role of a "vitamin role", or is it a reference to a vitamin role that some material entity bears that the authors are describing?

At moment vitamins sure sounds like material entities:
ChEBI: "vitamin": Any micronutrient that **is an organic compound**. The term "vitamines" (from vita + amines) was coined in 1912 by Casimir Funk, who believed that **these compounds** were amines.

ChEBI: "B vitamin": Any of the group of eight water-soluble vitamins originally thought to be a single compound (vitamin B) that play important roles in cell metabolism. The group comprises vitamin B1, B2, B3, B5, B6, B7, B9, and B12. (Around 20 other compounds were once thought to be B vitamins but are no longer classified as such.

etc.

**Comment by cmungall** [comment]
A few minority opinions:

I agree with @amalik01, I think that adding the string "role" just to satisfy ontologically precise text mining pipelines may be over-ontologization. The primary users of ontologies are not formal ontologists, most people say "food component" and "vitamin". A few decades ago GO affixed "activity" to the end of all catalytic activities in the ontology for similar ontological correctness reasons. Most members of the GOC now regard this as a mistake. And it's actually *worse* for text mining, because text is more likely to just say "protein kinase".

However, we do expect some overall coherency and consistency from ontologies, CHEBI is an odd state now where there was some half-hearted commitment to use BFO role a long time ago that was never really followed through on, with the labels and definitions being as if they are materials, as @ddooley correctly points out.

My minority opinion here was that CHEBI committing to formal ontological roles was a mistake, this has just caused confusion. I actually think it's perfectly fine to use definitions that are more like standard definitions for vitamin, pesticide, food component that non-ontologists would use. (They can still go in a separate hierarchy and be connected via non-isa relations of course). (if this is bothersome to formal ontologists I recommend the classic from Schulz et al https://pubmed.ncbi.nlm.nih.gov/21624161/, which is about diseases but applies here too)
